xbmc/pvr/recordings/PVRRecordings.cpp

```diff
@@ -135,18 +134,17 @@ std::vector<std::shared_ptr<CPVRRecording>> CPVRRecordings::GetAll() const
   std::vector<std::shared_ptr<CPVRRecording>> recordings;
 
   std::unique_lock lock(m_critSection);
-  std::transform(m_recordings.cbegin(), m_recordings.cend(), std::back_inserter(recordings),
-                 [](const auto& recordingEntry) { return recordingEntry.second; });
+  std::ranges::transform(m_recordings, std::back_inserter(recordings),
+                         [](const auto& recordingEntry) { return recordingEntry.second; });
```

Comment:
Suggestion:
```suggestion
  std::ranges::copy(std::views::values(m_recordings), std::back_inserter(recordings));
```
Same could be applied below.